www/src/assets/docs.svg

```diff
@@ -1,49 +1,11 @@
-<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 36 32" style="enable-background:new 0 0 36 32;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#CCB2E5;stroke:#663399;stroke-width:1.4173;stroke-miterlimit:10;}
-	.st1{fill:url(#SVGID_1_);}
-	.st2{fill:none;stroke:url(#SVGID_2_);stroke-width:1.4173;stroke-miterlimit:10;}
-	.st3{fill:none;stroke:url(#SVGID_3_);stroke-width:1.4173;stroke-miterlimit:10;}
-	.st4{fill:#FFFFFF;stroke:#9D7CBF;stroke-width:1.4173;stroke-miterlimit:140;}
-	.st5{fill:#663399;}
-	.st6{fill:url(#SVGID_4_);}
-</style>
-<g>
-	<path class="st0" d="M10,7c-1.6,0-3,1.4-3,3v2h10"/>
-	
-		<linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="41.5" y1="-55" x2="41.5" y2="-37" gradientTransform="matrix(1 0 0 1 -24 62)">
-		<stop  offset="0" style="stop-color:#FFFFFF"/>
-		<stop  offset="0.1799" style="stop-color:#FFFEFC"/>
-		<stop  offset="0.3245" style="stop-color:#FFFAF2"/>
-		<stop  offset="0.4568" style="stop-color:#FFF3E1"/>
-		<stop  offset="0.5818" style="stop-color:#FFEAC9"/>
-		<stop  offset="0.7017" style="stop-color:#FFDEAA"/>
-		<stop  offset="0.8177" style="stop-color:#FFD084"/>
-		<stop  offset="0.9285" style="stop-color:#FFBF59"/>
-		<stop  offset="1" style="stop-color:#FFB238"/>
-	</linearGradient>
-	<path class="st1" d="M10,7c1.6,0,3,1.4,3,3v12c0,1.6,1.3,3,3,3s3-1.4,3-3h6V10c0-1.6-1.4-3-3-3H10z"/>
-	<linearGradient id="SVGID_2_" gradientUnits="userSpaceOnUse" x1="17.8543" y1="22" x2="17.8543" y2="6.2913">
-		<stop  offset="0" style="stop-color:#9D7CBF"/>
-		<stop  offset="1" style="stop-color:#663399"/>
-	</linearGradient>
-	<path class="st2" d="M10,7h12c1.6,0,3,1.4,3,3v12"/>
-	<linearGradient id="SVGID_3_" gradientUnits="userSpaceOnUse" x1="13" y1="25.7087" x2="13" y2="6.2913">
-		<stop  offset="0" style="stop-color:#9D7CBF"/>
-		<stop  offset="1" style="stop-color:#663399"/>
-	</linearGradient>
-	<path class="st3" d="M16,25c-1.7,0-3-1.4-3-3V10c0-1.6-1.4-3-3-3"/>
-	<path class="st4" d="M16,25h9c1.6,0,3-1.4,3-3v-1h-9v1C19,23.6,17.6,25,16,25z"/>
-</g>
-<path class="st5" d="M17,10.5c0.6,0,1,0.4,1,1s-0.4,1-1,1c-0.6,0-1-0.4-1-1C16,10.9,16.6,10.5,17,10.5z"/>
-<path class="st5" d="M21.2,10.6c0.6,0,1,0.4,1,1s-0.4,1-1,1c-0.6,0-1-0.4-1-1C20.3,11.1,20.6,10.6,21.2,10.6z"/>
-<linearGradient id="SVGID_4_" gradientUnits="userSpaceOnUse" x1="19.2503" y1="14.5" x2="19.2503" y2="18.3161">
-	<stop  offset="0" style="stop-color:#9D7CBF"/>
-	<stop  offset="1" style="stop-color:#663399"/>
-</linearGradient>
-<path class="st6" d="M22.2,14.5c0,0,0,1.1,0,1.3c0,0.6-0.3,1.1-0.8,1.6c-0.5,0.5-1.3,0.9-2.2,0.9s-1.7-0.4-2.2-0.9
-	c-0.5-0.5-0.8-1-0.8-1.6c0-0.3,0-1.3,0-1.3C17.9,14.5,20.6,14.5,22.2,14.5z"/>
+<svg class="svg-icon-docs" x="0px" y="0px" viewBox="0 0 36 32" style="enable-background:new 0 0 36 32;">
```

Comment:
You can also just make these React components, which might be preferable to separate SVG files?

(Also I know this isn't _new_ in this PR 😅 )

Reply:
This approach would also mean we can drop the dependency on `react-svg-inline` which would be nice!

Reply:
Yep, def. interested in trying that — in fact I was very close to doing so, had https://svgr.now.sh/ opened already. ;)

Let me drop react-svg-inline now, not using any of it's features anyway!